maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -1560,6 +1566,18 @@ private void convertGroupParameters()
         }
     }
 
+    private void convertJunitEngineParameters()
+    {
+        if ( this.getJunitIncludeEngine() != null )
+        {
+            getProperties().setProperty( ProviderParameterNames.JUNIT_INCLUDE_ENGINE_PROP, this.getJunitIncludeEngine() );
+        }
+        if ( this.getJunitExcludeEngine() != null )
+        {
+            getProperties().setProperty( ProviderParameterNames.JUNIT_EXCLUDE_ENGINE_PROP, this.getJunitExcludeEngine() );
```

Comment:
Here can be the static import and the line would become shorter.

Reply:
I used the same format like it was for TESTNG_GROUPS_PROP but if you want I can follow your approach?